dimod/binary/binary_quadratic_model.py

```diff
@@ -1823,12 +1824,12 @@ class Float64BQM(BQM, default_dtype=np.float64):
     pass
 
 
-def Binary(label: Variable, bias: Bias = 1,
+def Binary(label: Variable = uuid.uuid4(), bias: Bias = 1,
```

Comment:
I believe this will assign the same label to each variable (determined when this module is imported).

Reply:
I'll add this test and fix this, thanks.